src/admin/content-block/helpers/generators/heading.ts

```diff
@@ -6,13 +6,13 @@ import {
 	DefaultContentBlockState,
 	HeadingBlockComponentState,
 } from '../../../shared/types';
-import { GET_HEADING_TYPE_OPTIONS } from '../../content-block.const';
+import { GET_FULL_HEADING_TYPE_OPTIONS } from '../../content-block.const';
 
 import { ALIGN_FIELD, BLOCK_FIELD_DEFAULTS, BLOCK_STATE_DEFAULTS, TEXT_FIELD } from './defaults';
 
 export const INITIAL_HEADING_COMPONENTS_STATE = (): HeadingBlockComponentState => ({
 	children: '',
-	type: 'h2',
+	type: 'h1',
```

Comment:
mss moeten we de default op H2 laten staan, zodat gebruikers expliciet moeten kiezen voor een h1 als ze dat wensen. Mogelijks vermijdt dit dat er 2 h1's toegevoegd worden aan 1 pagina?